tasks/components/CreateProfileTask.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { isNil } from 'ramda'
+import Profile from '../../agents/components/Profile'
+
+export default (props) => {
+  const { actions, currentAgent } = props
+
+  if(isNil(currentAgent)) {
+    return null
+  }
+
+  return <Profile
```

Comment:
obviously need buy-in from the rest of the team, but my current stack opinion is to use `hyperscript` and `recompose` to make pure functional components using only functions, on any new code or when we update old code.

why hyperscript? it makes us less dependent on a build step (since JSX will never be included in JavaScript), plus JSX is compiled to hyperscript anyways (`React.createElement` is actually `h(tagOrComponent, props, children)`) so it's more apparent why it works how it does, plus we are already using functions as the building block for the rest of our code.